lib/util/stream_utils.js

```diff
@@ -299,6 +299,16 @@ shaka.util.StreamUtils = class {
       }
     }
 
+    // |variant.audio.frameRate| can be undefined, which breaks
```

Comment:
This should say "variant.audio.channelsCount"

Reply:
Fixed!